db/migrate/20230430025441_add_archive_to_families.rb

```diff
@@ -0,0 +1,21 @@
+class AddArchiveToFamilies < ActiveRecord::Migration[7.0]
+  def up
+    add_column :families, :archived, :boolean
+    change_column_default :families, :archived, false
+  end
+
+  def down
+    remove_column :families, :archived
+  end
+end
+
+class BackfillAddArchiveToFamilies < ActiveRecord::Migration[7.0]
```

Comment:
It's really unintuitive to have multiple migrations in the same file. Pretty sure the second one wouldn't even run. Better to create two separate migrations.

But... do you need the second migration? Honestly the number of families is so tiny that I think you can ignore the `safe_migrations` warnings and just add the default with the column (and set it to `null: false`) at the same time.

Reply:
Nice! I will ignore that warning! Thank you for the feedback